### ISSUE TITLE
[3.0.3] HUSKY_SKIP_INSTALL=1 doesn't work

### ISSUE THREAD

**Issue by yekver** [typicode/husky#563]
```bash
Step 7/20 : RUN HUSKY_SKIP_INSTALL=1 npm ci
 ---> Running in 9ydfa87dc7df

> husky@3.0.3 install /usr/src/node_modules/husky
> node husky install

husky > Setting up git hooks
spawnSync git ENOENT
husky > Failed to install

> husky@3.0.3 postinstall /usr/src/node_modules/husky
> opencollective-postinstall || true

Thank you for using husky!
If you rely on this package, please consider supporting our open collective:
> https://opencollective.com/husky/donate

```

**Comment by yekver** [comment]
Maybe related to https://github.com/npm/cli/commit/8b43c962498c8e2707527e4fca442d7a4fa51595

**Comment by typicode** [comment]
Thanks for the report, it should be fixed in `3.0.4`.
